R/vars_manager.R

```diff
@@ -227,6 +227,10 @@ get_matrix_var <- function(
     ))
 }
 
+
+#' @importFrom dplyr arrange %>%
+#' @importFrom rlang parse_exprs
+#' @importFrom tibble rownames_to_column column_to_rownames
```

Comment:
It looked like this was what roxygen2 wanted - I added these here, and added them to the DESCRIPTION file in the Imports section